resources/views/pageRedirect.html

```diff
@@ -0,0 +1,4 @@
+<head>
+    <p>Redirecting to Dataspace application...</p>
+    <meta http-equiv="refresh" content="0; URL=https://dataspace.cavd.org/cds/CAVD/app.view" />
```

Comment:
This will be confusing on staging servers and dev machines. Can we use a relative URL, like this?

```suggestion
    <meta http-equiv="refresh" content="0; URL=/cds/CAVD/app.view" />
```

Reply:
It won't work on dev machines anyway, since the project and folder names are not the same.
In any case, Dataspace will be only using/care for this redirect for their production - it won't be set on their staging